extensions/github-authentication/package.json

```diff
@@ -15,8 +15,7 @@
   ],
   "extensionKind": [
     "ui",
-    "workspace",
-    "web"
```

Comment:
why do we need to make these edits to the vscode extensions?  if it's needed then this is fine, but if this is optional I would avoid updating any files in the vscode upstream code to minimize merge conflicts in the vscode source code refreshes.

Reply:
@kburtram, 'web' extension was giving me hygiene errors as it is not allowed in the latest version of vsce.